js/src/ui/AccountCard/accountCard.js

```diff
@@ -36,15 +37,16 @@ export default class AccountCard extends Component {
   };
 
   static defaultProps = {
-    disableAddressClick: false
+    disableAddressClick: false,
+    showPassword: false
```

Comment:
That's so bizarre, my local branch doesn't have that code... hopefully the next up fixes that.